src/Sulu/Bundle/WebsiteBundle/Tests/Unit/Resolver/RequestAnalyzerResolverTest.php

```diff
@@ -112,20 +116,23 @@ public function testResolve()
         $requestAnalyzer->getResourceLocator()->willReturn('/search');
         $requestAnalyzer->getPortal()->willReturn($portal);
         $requestAnalyzer->getPortalInformation()->willReturn($portalInformation->reveal());
+        $requestAnalyzer->getSegment()->willReturn($segment);
 
         $result = $this->resolver->resolve($requestAnalyzer->reveal());
         $this->assertEquals(
             [
                 'request' => [
                     'webspaceKey' => 'sulu_io',
                     'webspaceName' => 'Sulu',
+                    'segmentKey' => 's',
                     'portalKey' => 'sulu_io_portal',
                     'portalName' => 'Sulu Portal',
                     'defaultLocale' => 'de',
                     'portalUrl' => 'sulu.io/de',
                     'resourceLocatorPrefix' => '/de',
                     'resourceLocator' => '/search',
                 ],
+                'segmentKey' => 's',
```

Comment:
Not totally sure about this one... The `webspaceKey` is also available under the `request` key and on the top level in the twig template... But that's because the `webspaceKey` is returned by the `RequestAnalyzerResolver` and the `StructureResolver`. But the `StructureResolver` does not know about the segment, so I can't add it there as well...

Reply:
is that evertime the same? if yes why not removing one of them - if not we need another name - like `currentSegmentKey` and `documentSegmentKey` or something like this

Reply:
I decided to put it in both places, because personally I would expect the `segmentKey` at the same place where I get the `webspaceKey`, since they are related. And since the `webspaceKey` is in two different places, I've also put the segment in two different places.

Prefixing `segmentKey` with `current` or `document` does not make a lot of sense to me, because none of the other twig variables have any of these prefixes.

Reply:
think i would only add it to the `request` array. two places for the same information feel strange to me. we cant fix it for the `webspaceKey` now, but we also dont need to repeat it 🙂